src/maptiler/cloud_cli/base.py

```diff
@@ -142,3 +141,19 @@ def handle_response_errors(response: Response):
         print(error)
         print(error.response.json()["errors"][0]["message"])
         exit()
+
+def task_request(http: requests.Session, url: str) -> dict:
+    try:
+        response = http.get(url)
+        handle_response_errors(response)
+        response_data = response.json()
+        retries = 0
+        return response_data
+    except ConnectionResetError:
+        print("Connection reset by peer, retrying")
+        if retries <= MAX_RETRIES:
+            retries += 1
+            sleep(3 * retries)
+            task_request(http, url)
+        else:
+            raise
```

Comment:
1. Variables in Python are local by default, so setting `retries` to zero will create a new local variable.
2. DO NOT use global variables like this. The code is extremely confusing. Turn it into a normal loop.
3. Use `click.echo()` instead of `print()`, like the rest of the code.

```python
retries = 0
while True:
    try:
        ...
    except ...:
        retries += 1
        if retries <= MAX_RETRIES:
            sleep(...)
        else:
            raise


```